python/python/bystro/search/utils/messages.py

```diff
@@ -22,6 +27,32 @@ class IndexJobCompleteMessage(CompletedJobMessage, frozen=True, kw_only=True):
     results: IndexJobResults
 
 
+class BinomialMafFilter(
+    Struct, frozen=True, tag="binomMaf", tag_field="key", forbid_unknown_fields=True
+):
+    """
+    Example: {'critValue': '.05', 'estimates': ['gnomad.genomes.af', 'gnomad.exomes.af'],
```

Comment:
Can you add a brief description of what the binomial maf filter is useful for here? Especially since it's not as common as the HWE step in genomics workflows.

Reply:
Done!